operators/derive/to_mapbox.py

```diff
@@ -15,34 +17,46 @@ def geo_data_flow():
         DF.add_field(
             'record',
             'object',
-            lambda r: {k: str(v) for k, v in r.items()},
-            resources=['geo_data'],
-        ),
-        DF.add_field(
-            'offset',
-            'array',
-            lambda r: (0, 0),
-            constraints={'maxLength': 2},
+            lambda r: {
+                k: v if not k == 'branch_geometry' or v is None else (float(v[0]), float(v[1]))
+                for k, v in r.items()
+            },
             resources=['geo_data'],
         ),
-        helpers.unwind('response_categories', 'response_category'),
+        helpers.unwind('response_categories', 'response_category', resources=['geo_data']),
         # some addresses not resolved to points, and thus they are not useful for the map.
         DF.filter_rows(lambda r: not r['branch_geometry'] is None, resources=['geo_data']),
         DF.join_with_self(
             'geo_data',
-            ['card_id'],
+            ['branch_geometry', 'response_category'],
             fields=dict(
-                card_id=None,
                 branch_geometry=None,
-                offset=None,
                 response_category=None,
-                outer_situations={'name': 'situations', 'agreggate': 'set'},
-                outer_responses={'name': 'responses', 'agreggate': 'set'},
-                records={'name': 'record', 'agreggate': 'set'},
+                situations_at_point={'name': 'situations', 'aggregate': 'array'},
+                responses_at_point={'name': 'responses', 'aggregate': 'array'},
+                records={'name': 'record', 'aggregate': 'array'},
             ),
         ),
+        DF.add_field(
+            'offset',
+            'array',
+            lambda r: (0, 0),
+            constraints={'maxLength': 2},
+            resources=['geo_data'],
+        ),
         DF.set_primary_key(['branch_geometry', 'response_category']),
-        DF.rename_fields({'outer_situations': 'situations', 'outer_responses': 'responses'}),
+        DF.add_field(
```

Comment:
let's simplify this and for the outer situations/responses just have a simple list of the ids, without the names.

Reply:
i.e. `['human-situations:age-group:adults', ....]`

Reply:
ok

Reply:
@akariv updated.